Admin/RouteAdmin.php

```diff
@@ -174,7 +174,7 @@ protected function configureFieldsForDefaults($dynamicDefaults)
             $defaults['_format'] = array('_format', Sf2CompatUtil::getFormTypeName('text'), array('required' => true));
         }
         if ($route && $route->getOption('add_locale_pattern')) {
-            $defaults['_locale'] = array('_format', Sf2CompatUtil::getFormTypeName('text'), array('required' => false));
+            $defaults['_locale'] = array('_locale', Sf2CompatUtil::getFormTypeName('text'), array('required' => false));
```

Comment:
@WouterJ found one surprising thing here. i think this is a copy-paste mistake that happened in master while refactoring, right?